frontend/src/lib/api.ts

```diff
@@ -52,7 +53,10 @@ export * from "./api/request";
 export * from "./api/types";
 
 export function useThisUser(): User | AnonymousUser | undefined {
-    const { data: user, error } = useSWR<AnonymousUser | User>("/user", get);
+    const { data: user, error } = useSWRImmutable<AnonymousUser | User>(
```

Comment:
Usernames can change, though it is rare.. wondering if anything else can change that might be important...

Reply:
User endpoint is `mutate`'d when a user logs out, so this is ok.